combining_caribou_teeth.R

```diff
@@ -1,17 +1,20 @@
-sample01A <- read.csv("01A.csv", header = FALSE)
-colnames(sample01A) <- c("time", "Sr")
 
-sample01B <- read.csv("01B.csv", header = FALSE)
-colnames(sample01B) <- c("time", "Sr")
+sampler <- function(fileNumber, fileCharacter) {
+  fileName <- sprintf("%02d%s.csv", fileNumber, fileCharacter)
+  sample <- read.csv(fileName)
+  colnames(sample) <- c("time", "Sr")
+  return(sample)
+}
 
-sample01C <- read.csv("01C.csv", header = FALSE)
-colnames(sample01C) <- c("time", "Sr")
 
-sample01 <- rbind(sample01A, sample01B, sample01C)
-plot(sample01, type = "p", pch = 20, cex = 0.5)
+for (fileCount in 1:15) {
```

Comment:
This will loop through the numbers 1 to 15 (I believe inclusively, but I did not verify that)